packages/engine/README.md

```diff
@@ -29,14 +34,7 @@ yarn add @cesium/engine
 
 ## Usage
 
-```js
-import * as Cesium from "@cesium/engine";
-import "@cesium/engine/Source/Widget/CesiumWidget.css";
-
-const cesiumWidget = new Cesium.CesiumWidget("cesiumContainer");
-```
-
-Or, import individual modules to benefit from tree shaking optmtimizations through most build tools:
+Import individual modules to benefit from tree shaking optimizations through most build tools:
 
 ```js
 import { CesiumWidget } from "@cesium/engine";
```

Comment:
`CesiumWidget` as part of `engine` rather than `widgets` can be confusing at first glance. Could we use another class (maybe `Cartesian3`) as the example here? Is `CesiumWidget` the most common use case for people switching from `cesium` to `@cesium/engine`?

Reply:
I agree that the naming is indeed confusing, however this is the most common use case. I think the naming issue should likely be addressed outside of this PR.

Reply:
OK, makes sense